src/main/java/Main.java

```diff
@@ -1,193 +1,36 @@
-import controllers.Tags.*;
-import controllers.Tasks.*;
-import entities.*;
-import gateways.Tags.TagDataAccessInterface;
-import gateways.Tags.TagDatabaseGateway;
-import gateways.Tasks.TaskDataAccessInterface;
-import gateways.Tasks.TaskDatabaseGateway;
+import entities.Task;
+import entities.User;
+import gateways.DataAccessInterface;
+import gateways.TaskDatabaseGateway;
 import gateways.UserDatabaseGateway;
-import presenters.*;
-import screens.TaskList.*;
+import screens.MainInvitationScreen.MainInvitationMethod;
 import screens.WelcomeScreen;
 import services.CurrentUserService;
-import useCases.Tags.*;
-import useCases.Tasks.*;
 
 import javax.swing.*;
-import java.awt.*;
 import java.io.IOException;
 
-
 public class Main {
-
-    static User user = new User("User", "Password");
+    static DataAccessInterface<User> userGateway; //
+    static DataAccessInterface<Task> taskGateway; //
+    static CurrentUserService current_user_service;
 
     public static void main(String[] args) throws IOException {
 
-        // Instantiating all Factories
-        TaskFactory taskFactory = new TaskFactory();
-        TagFactory tagFactory = new TagFactory();
-
-        // Instantiating all Presenters
-        TaskPresenter taskPresenter = new TaskResponseFormatter();
-        TaskInfoPresenter taskInfoPresenter = new TaskInfoResponseFormatter();
-        TagPresenter tagPresenter = new TagResponseFormatter();
-
-        // Instantiating all Services
-        CurrentUserService currentUserService = new CurrentUserService();
-
-        // Instantiating all Database Gateways
-        UserDatabaseGateway userDatabaseGateway = new UserDatabaseGateway("testDatabase.ser");
-
-        User user = userDatabaseGateway.get("User");
```

Comment:
Please make another commit with Leon's stuff un-deleted. Screwing up Main will mean the whole program doesn't run.